Remove usages of lane departure state

chezzdev opened the pull request:
Resolves #59

dersim-davaod commented:
@chezzdev, not clear at first sight:
Should we update/remove `laneDepartureWarning` from `AlertPlayer.swift`?
Should we remove `lane_departure_warning.wav` from resources?
Should we remove `lane-departure-notification` pic from asset catalog?

General question: should we regenerate resources with `swiftgen` then?

chezzdev commented:
> Please remove it from `MockContainerPresenter.swift`.

> @chezzdev, not clear at first sight:
> Should we update/remove `laneDepartureWarning` from `AlertPlayer.swift`?
> Should we remove `lane_departure_warning.wav` from resources?
> Should we remove `lane-departure-notification` pic from asset catalog?

Good catch. Removed.

> General question: should we regenerate resources with `swiftgen` then?

`SwiftGen` should do it automatically. I've added resources as an input to `SwiftGen` phase so that in theory it should run every time resources are changed. In practice it runs every build now but I guess it's acceptable.
